spec/sendsms_spec.rb

```diff
@@ -0,0 +1,6 @@
+
+describe 'sendsms' do
+  subject(:sendsms) {described_class.new(order, total)}
+  let(:order) { double() }
+  let(:total) { double() }
```

Comment:
Style/MethodCallParentheses: Do not use parentheses for method calls with no arguments. (https://github.com/bbatsov/ruby-style-guide#no-args-no-parens)